CMakeLists.txt

```diff
@@ -1,13 +1,11 @@
-cmake_minimum_required(VERSION 2.8.3)
-project(ros_environment)
-find_package(catkin REQUIRED)
-
-catkin_package()
+cmake_minimum_required(VERSION 3.5)
+project(ros_environment NONE)
+find_package(ament_cmake_core REQUIRED)
 
 if(CMAKE_HOST_UNIX)
-  set(shells "sh")
+  set(shell "sh")
 else()
-  set(shells "bat")
+  set(shell "bat")
```

Comment:
I know the catkin and ament API are different but it would be great to have the same variable names in both branches to keep diff at a minimum

Reply:
I can rename the variable in the ROS 1 branch but the diff is irrelevant since each distro needs its own branch anyway.

Reply:
> I can rename the variable in the ROS 1 branch but the diff is irrelevant since each distro needs its own branch anyway.

Agreed we will never merge these branches together. It just seem easier to compare what's different in ROS1 and ROS 2 if we use the same variable names but that's not a blocker for this PR to be merged